docs/supervisor-datamgr.md

```diff
@@ -0,0 +1,72 @@
+
+## Setting up Backup Network
+It is recommended that the kubernetes backup/restore traffic be separated from the vSphere management network on a
+workload cluster. A backup network can be configured as an NSX network or traditional TDS network. We can add a VMkernel
+NIC on each host in the cluster and set the vSphereBackupNFC on that NIC. This enables backup network traffic to be sent
+through that NIC. If the vSphereBackupNFC is not enabled, the backup traffic will be sent on the management network.
+
+More details can be found in the [vSphere documentation](https://code.vmware.com/docs/12628/virtual-disk-development-kit-programming-guide/GUID-5D166ED1-7205-4110-8D72-0C51BB63CC3D.html).
+
+Here are some ways to setup a Backup Network with vSphereBackupNFC tag enabled on a NSX setup.
+
+If using the existing vSphere Distributed Switch for the cluster (Also for VCF environment):
+1. Go to vCenter Menu -> Networking
+2. On the cluster Distributed switch, create a DistributedPortGroup called BackupNetwork
+3. Add VMkernel adapter to the DistributedPortGroup BackupNetwork, attach all the ESXi hosts in the cluster and 
+enable the "vSphereBackupNFC" tag
+
+If there is a free physical network adpter on each of the cluster nodes
+1. On vCenter UI - go to ESXi server -> Configure -> Networking -> Virtual switches -> Add Networking
+2. Create a new Standard switch on the free network adapter on each ESXi host in the cluster, with a label BackupNetwork
+3. Add VMkernel port to each Standard switch with vSphereBackupNFC tag
+
+## Data Manager Virtual Machine install
+TODO: Add location of OVA
+
+The Data Manager Virtual Machine ova can be downloaded from the files folder. TODO: Add location
+
+It is recommended to power on the Data manager VM after enabling the velero-vsphere service and installing velero + vSphere
+plugin in the workload cluster.
+
+1. Install the DataManager VM in the vSphere cluster on the backup network. Customize any configuration based on your environment.
+2. Configure the input parameters for the VM. On the VC UI go to VM -> Edit Settings -> VM Options -> Advanced -> Edit Configuration Parameters
+   - guestinfo.cnsdp.wcpControlPlaneIP
+   - guestinfo.cnsdp.vcAddress
+   - guestinfo.cnsdp.vcUser
+   - guestinfo.cnsdp.vcPassword
+   - guestinfo.cnsdp.vcPort
+   - guestinfo.cnsdp.veleroNamespace
+   - guestinfo.cnsdp.datamgrImage (if not configured, will use the image from dockerhub vsphereveleroplugin/data-manager-for-plugin:1.1.0)
+   - guestinfo.cnsdp.updateKubectl (default false, to avoid kubectl from wcp master on every VM restart)
+3. Power On the Data Manager VM
+
+## Debugging Data Manager issues, container image
+- Login to Data Manager VM
+  - Power on the VM and login - ssh or web console
+  - First time root password is "changeme". Update it.
+- Check velero data manager status and logs
+  - "docker ps" to confirm data manager container is running.
+  - "docker logs velero-datamgr" to get container logs
```

Comment:
docker command does not work any more on Supervisor Cluster?

Reply:
This command is to be executed inside the data manager VM.

Reply:
Right.